core/src/main/resources/error/error-classes.json

```diff
@@ -96,10 +151,22 @@
     "message" : [ "Unrecognized SQL type %s" ],
     "sqlState" : "42000"
   },
+  "UNSUPPORTED_ARRAY_ELEMENT_TYPE_BASED_ON_BINARY" : {
+    "message" : [ "Unsupported array element type %s based on binary" ],
+    "sqlState" : "0A000"
+  },
+  "UNSUPPORTED_JDBC_TYPE" : {
+    "message" : [ "Unsupported type %s" ],
+    "sqlState" : "0A000"
+  },
   "UNSUPPORTED_LITERAL_TYPE" : {
     "message" : [ "Unsupported literal type %s %s" ],
     "sqlState" : "0A000"
   },
+  "UNSUPPORTED_OPERATION_FOR_DATA_TYPE" : {
```

Comment:
This does not reflect the error message. Can you improve the error message for this?